config.example.yaml

```diff
@@ -0,0 +1,19 @@
+logger:
+  level: "debug"
+
+nats:
+  streams:
+    - name: "test"
+      subjects: ["test.*"]
```

Comment:
i think we should mention that subject can be optional. Also why subject is considered a list of strings? we need only one test subject

Reply:
Thank you for your feedback.

I initially considered multiple streams with multiple subjects to handle complex test cases. However, I've simplified it to use a single test subject for now.